api/standard/clause_8_ogcapi_processes.adoc

```diff
@@ -21,81 +21,90 @@ Exchange Model in GeoJSON.
-.2+|Jobs .2+|`/processes/routing/jobs` |GET |<<wps_jobs>>
-|POST |<<wps_new_job>>
-|Job |`/processes/routing/jobs/{jobId}` |GET |<<wps_job_status>>
-|Result |`/processes/routing/jobs/{jobId}/result` |GET |<<wps_result>>
+|Landing page |`/` |GET |<<oaproc_landing_page>>
+|Conformance declaration |`/conformance` |GET |<<oaproc_conformance>>
+|Processes |`/processes` |GET |<<oaproc_processes>>
+|Process |`/processes/routing` |GET |<<oaproc_process>>
+.2+|Jobs .2+|`/jobs` |GET |<<oaproc_jobs>>
+|POST |<<oaproc_new_job>>
+|Job |`/jobs/{jobId}` |GET |<<oaproc_job_status>>
+|Result |`/jobs/{jobId}/results` |GET |<<oaproc_result>>
 !===
 
-[[wps_landing_page]]
+NOTE: Proposal being discussed ( https://github.com/opengeospatial/ogcapi-processes/issues/124 ) to POST execution request to `/processes/{processId}` instead of to `/jobs`.
+If we manage to fully harmonize the Processes / `/routes` approach, `/routes` could be an alias to `/processes/routing` supporting same POST requests.
+
+[[oaproc_landing_page]]
 === Landing page
 
 See <<landing_page>>, except that the "data" link references `/processes`.
 
-[[wps_conformance]]
+[[oaproc_conformance]]
 === Conformance declaration
 
-The following is an example of the conformance declaration for the WPS option
-that implements all requirements classes. Note that the following classes
-of the Routing API option are not available in the WPS option as OGC API Processes
-does not support the required capabilities:
+The following is an example of the conformance declaration for the _OGC API - Processes_ option
+that implements all requirements classes. The ResultSet class has no equivalent functionality with _OGC API - Processes_.
 
-* Delete route
-* Callback
-* Result set
-* Synchronous execution
+Suggestion: Define a single specific conformance class for the OGC API - Processes variants, and re-use the existing Routing conformance classes?
 
-[[example_wps_cc]]
+[[example_oaproc_cc]]
 .Conformance declaration
 =================
 [source,JSON]
 ----
 {
   "conformsTo": [
-    "http://www.opengis.net/spec/WPS/2.0/req/service/binding/restjson/core",
-    "http://www.opengis.net/spec/WPS/2.0/req/service/binding/restjson/oas30",
-    "http://www.opengis.net/spec/ogcapi-routes-1/1.0/conf/wps-intermediate-waypoints",
-    "http://www.opengis.net/spec/ogcapi-routes-1/1.0/conf/wps-max-height",
-    "http://www.opengis.net/spec/ogcapi-routes-1/1.0/conf/wps-max-weight",
-    "http://www.opengis.net/spec/ogcapi-routes-1/1.0/conf/wps-obstacles",
-    "http://www.opengis.net/spec/ogcapi-routes-1/1.0/conf/wps-routing-engine",
-    "http://www.opengis.net/spec/ogcapi-routes-1/1.0/conf/wps-routing-algorithm",
-    "http://www.opengis.net/spec/ogcapi-routes-1/1.0/conf/wps-source-dataset",
-    "http://www.opengis.net/spec/ogcapi-routes-1/1.0/conf/wps-time",
+    "http://www.opengis.net/spec/ogcapi-common-1/1.0/conf/json",
+    "http://www.opengis.net/spec/ogcapi-common-1/1.0/conf/oas30",
```

Comment:
If it conforms with this, doesn't it also need to conform to http://www.opengis.net/spec/ogcapi-processes-1/1.0/conf/oas30 ?

Reply:
Why does Processes need its own OpenAPI 3.0 conformance class, if there is a dependency to Common?

Reply:
I have raised the issue of separate common-inherited oas30 conformance class with Processes.